src/solver/expressions/include/antares/solver/expressions/nodes/BinaryNode.h

```diff
@@ -27,12 +27,37 @@ namespace Antares::Solver::Nodes
 class BinaryNode: public Node
 {
 public:
+    /**
+     * @brief Constructs a binary node with the specified left and right operands.
+     *
+     * @param left The left operand.
+     * @param right The right operand.
```

Comment:
Note: BinaryNode(n1, n2) and BinaryNode(n2, n1) are not equivalent.

Reply:
updated